cli/pcluster/dcv/connect.py

```diff
@@ -52,13 +59,35 @@ def dcv_connect(args):
         DCV_SHARED_DIR=shared_dir,
     )
 
-    # Connect by ssh to the master instance and prepare DCV session
     try:
-        LOGGER.debug("SSH command: {0}".format(cmd))
-        output = _check_command_output(cmd)
+        url = retry(_retrieve_dcv_session_url, func_args=[cmd, args.cluster_name, master_ip], attempts=4)
```

Comment:
Why `retry()` function and not `@retry` decorator? Also why 4 attempts?

Reply:
I don't want to include the retrying library in this patch release. We can decide to add it in the next round.

Reply:
Each round has an internal retry of 5 seconds (see pcluster_dcv_authenticator.py code) so a retry of 4 means 4*5 = 20 seconds